ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly.yaml

```diff
@@ -1704,6 +1704,33 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-gcp-ipi-disk-encryption
+- as: gcp-ipi-oidc-rt-fips-amd-regen-cert-f14
```

Comment:
same with https://github.com/openshift/release/pull/54930/files#r1699460724

Reply:
Sure, I will wait your PR get merged, then rebase mine, on that time, I will update them.